create.js

```diff
@@ -29,9 +29,14 @@ async function run() {
 
     core.info("Generating config...");
 
+    var server_name = core.getInput("serverName");
+    if (server_name == "") {
+       server_name = "localhost"
```

Comment:
We specify `public_baseurl` on line 47 - We should probably match the URL there?

This may cause clients to do weird things (like trying to connect to the provided server_name outside of GHA).